llvm/lib/CodeGen/MachineBasicBlock.cpp

```diff
@@ -1598,15 +1598,42 @@ MachineBasicBlock::getProbabilityIterator(MachineBasicBlock::succ_iterator I) {
   return Probs.begin() + index;
 }
 
+MachineBasicBlock::LivenessQueryResult
+MachineBasicBlock::computeRegisterLiveness(const TargetRegisterInfo *TRI,
+                                           MCRegister Reg,
+                                           const_iterator Before) const {
+  LivePhysRegs UsedRegs(*TRI);
```

Comment:
LiveRegUnits should always be preferred to LivePhysRegs. No new uses of LivePhysRegs should be introduced